lib/facility_factory.rb

```diff
@@ -0,0 +1,15 @@
+class FacilityFactory
+ def or_facility_creator(or_dmv_data)
+  or_dmv_info = []
+  or_dmv_data.each do |data|
+    or_dmv_info << Facility.new({
+      name: data[:title], 
+      address: data[:human_address], 
+      phone: data[:phone_number]})
+    require 'pry'; binding.pry
```

Comment:
Don't forget to remove your `prys`. Also, the more conventional way to split this out onto multiple lines is
```ruby
or_dmv_info << Facility.new({
     name: data[:title], 
     address: data[:human_address], 
     phone: data[:phone_number]
})
```